docs/pages/api-reference/liveblocks-react-lexical.mdx

```diff
@@ -258,6 +258,11 @@ function Toolbar() {
 
 ### FloatingThreads
 
+<Banner
+  title="Floating threads are available since version 2.3"
+  type="warning"
+/>
+
```

Comment:
```suggestion
```
Hey, we don't normally add banners for new features, as they'd be all over our documentation if we did! For example, we added about 25 new methods in 2.4.0.

Reply:
Oh okay I see. TBH I think it's a bit sad to not have this kind of information from a customer standpoint but I understand your point 🙂

Reply:
Yeah, I hear you! Perhaps if we had a more subtle way to write it down, we could do it.